content/en/faqitems/project-1-bottlerocket-and-firecracker.markdown

```diff
@@ -0,0 +1,16 @@
++++
+question = "What's the difference between Firecracker and Bottlerocket?"
+group = "Project"
++++
+
+Quite a lot!
+[Firecracker](https://firecracker-microvm.github.io/) is a *virtualization* technology and Bottlerocket is an *operating system*. From the [Firecracker FAQ](https://firecracker-microvm.github.io/#faq):
+
+> Firecracker is an alternative to QEMU that is purpose-built for running serverless functions and containers safely and efficiently, and nothing more.
```

Comment:
Alternatively: Firecracker really just runs micro VMs and talking about specifics of what's inside those micro VMs (containers, serverless functions) is actually what creates the confusion, since Bottlerocket is also built for running containers safely and efficiently, and nothing more.

The missing nuance here is that there are different ways to run containers:
1. using the Linux kernel primitives for isolation (cgroups, namespaces)
2. using userspace reimplementation of the Linux kernel API (gvisor)
3. using micro VMs with micro Linux kernels for each container (Firecracker)

Bottlerocket could support all of these ways with various trade-offs in terms of security and functionality, and today just supports the first one.